src/utils/Utility.ts

```diff
@@ -78,6 +78,18 @@ class Utility {
       .match(/^\d+(\.\d{0,1})?$/);
   };
 
+  static getUpper = (str: string) => {
+    if (typeof str === 'string') {
```

Comment:
why is this done? str will be string only right

Reply:
had added that just as a check, will remove it